doc/rst/tools/mason/mason.rst

```diff
@@ -8,8 +8,6 @@ mason
 Mason is Chapel's package manager
 
 
```

Comment:
A TOC might be nice, given how long this page is getting now, e.g.

```
.. contents:: Table of Contents
```

Reply:
Oh.  I didn't realize that the code you gave was all there was to it.  I was imagining having to create all sorts of RST links between the TOC and the sections.